include/irr/core/math/floatutil.h

```diff
@@ -142,15 +143,54 @@ union FloatIntUnion32
 	float f;
 };
 
-//! code is taken from IceFPU
 //! Integer representation of a floating-point value and the reverse.
 #ifdef __IRR_FAST_MATH
-#define IR(x)                           ((uint32_t&)(x))
-#define FR(x)                           ((float&)(x))
+IRR_FORCE_INLINE uint32_t& IR(float& x)
```

Comment:
we have `floatBitsToInt` and `intBitsToFloat` in glslFunctions.h so we can get rid of those later (i guess they're used somewhere so its not entirely trivial to remove them)

Reply:
go ahead, do it!